docs/articles_en/openvino_workflow/model_optimization_guide/weight_compression.rst

```diff
@@ -144,13 +149,62 @@ The table below shows examples of Text Generation models with different optimiza
      - INT4_SYM,group_size=64,ratio=0.8
      - 2.98
      - 8.0
+
+
+The following table shows accuracy metric in a data-aware 4-bit weight quantization setup measured on `Wikitext dataset <https://arxiv.org/pdf/1609.07843.pdf>`__.
+
+.. list-table::
+   :widths: 40 55 25 25
+   :header-rows: 1
+
+   * - Model
+     - Optimization
+     - Perplexity
+     - Model Size (Gb)
+   * - meta-llama/llama-7b-chat-hf
+     - FP32
+     - 11.87
+     - 10.3
+   * - meta-llama/llama-7b-chat-hf
+     - INT4_SYM,group_size=128,ratio=1.0,awq=True
+     - 5.07
+     - 2.6
+   * - stabilityai_stablelm-3b-4e1t
+     - FP32
+     - 11.87
+     - 10.3
+   * - stabilityai_stablelm-3b-4e1t
+     - INT4_SYM,group_size=64,ratio=1.0,awq=True
+     - 5.07
+     - 2.6
+   * - HuggingFaceH4/zephyr-7b-beta
+     - FP32
+     - 11.87
+     - 10.3
+   * - HuggingFaceH4/zephyr-7b-beta
+     - INT4_SYM,group_size=128,ratio=0.8
+     - 5.07
+     - 2.6
    
 
+Auto-tuning of weight compression parameters
+############################################
+
+The important question that may arise is how to find a configuration of weight compression parameters that is best suited to a particular model.
```

Comment:
@l-bat, please check the section about weight compression parameters tuning, consider adding extra info if needed.